iothub/device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/mqtt/MqttTwin.java

```diff
@@ -122,28 +122,13 @@ private String buildTopic(final IotHubTransportMessage message)
                     throw new IllegalArgumentException("Request Id is Mandatory");
                 }
 
-                int version = message.getVersion();
-                topic.append(AND);
-                topic.append(VERSION);
-                topic.append(version);
-                break;
-            }
-            case DEVICE_OPERATION_TWIN_SUBSCRIBE_DESIRED_PROPERTIES_REQUEST:
```

Comment:
This particular block would return the topic string to subscribe on, but this method is about building publish payloads, so we can remove this